openedx/features/enterprise_support/api.py

```diff
@@ -570,6 +571,17 @@ def get_enterprise_learner_data(user):
             return enterprise_learner_data['results']
 
 
+@enterprise_is_enabled()
+def get_enterprise_learner_data_from_db(user):
+    """
+    Query the database directly and use the same serializer that the api call would use to return the same results.
+    """
+    if user.is_authenticated:
+        queryset = EnterpriseCustomerUser.objects.filter(user_id=user.id)
+        serializer = EnterpriseCustomerUserReadOnlySerializer(queryset, many=True)
```

Comment:
One thought, this seems like it should be a `get` without serializing `many=True` though

Reply:
I still have this thought that it feels weird to filter and serialize with many for a single user. If it's for backwards compatibility might be nice to have a small comment about this for the future?

Reply:
forgot to respond to this - frequently enough, users are linked to multiple enterprises. i'm not sure a special callout is necessary, and this is what the serializer is doing behind the scenes with the api call, so even if it's a single entry, we want it returned as a list.

Reply:
gotcha, that makes sense